integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -489,6 +489,20 @@ public static ExecResult cleanup() throws Exception {
     return ExecCommand.exec(cmd);
   }
 
+  public static ExecResult cleanupDomainResources(String domainUid) throws Exception {
```

Comment:
There is already one in TestUilts.deleteWeblogicDomainResources, you can use that.

Reply:
updated and removed it from Base